test/unique.ts

```diff
@@ -39,8 +39,16 @@ test('Create user 1', async() => {
     let items = await table.scanItems()
     expect(items.length).toBe(3)
 
-    let unique = items.filter((item) => item.pk.S.indexOf('interpolated') >= 0)
-    let pk = unique[0].pk.S
+    let pk = (() => {
+        if (isV3()) {
+            let unique = items.filter((item) => (item.pk.S).indexOf('interpolated') >= 0)
+            return unique[0].pk.S
```

Comment:
Same as per previous comment. V3 and V2 should be returning identical data.

Reply:
https://github.com/sensedeep/dynamodb-onetable/pull/270#discussion_r800164728